frontend/components/PersonaCard.tsx

```
@@ -388,10 +388,10 @@ Remember: You are ${editFields.name}, not an AI assistant. Respond as this chara
         {/* Left Column - Basic Info */}
         <div className="space-y-4">
           <div>
-            <label className="block text-sm font-medium text-gray-700 mb-2">Background/Bio</label>
+            <label className="block text-sm font-medium text-gray-700 mb-2">Background</label>
             <Textarea
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)